basic-features.md

```diff
@@ -1,80 +1,136 @@
+| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            | Y            | Y            |
+| `UNION`, `UNION ALL`                                                                                     | Y            | Y            | Y            | Y            | Y            | Y            |
+| [`EXCEPT` and `INTERSECT` operators](/functions-and-operators/set-operators.md)                                                  | Y            | Y            | N            | N            | N            | N            |
+| `GROUP BY`, `ORDER BY`                                                                                   | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Window Functions](/functions-and-operators/window-functions.md)                                                                 | Y            | Y            | Y            | Y            | Y            | N            |
+| [Common Table Expressions (CTE)](/sql-statements/sql-statement-with.md)                                  | Y            | N            | N            | N            | N            | N            |
+| `START TRANSACTION`, `COMMIT`, `ROLLBACK`                                                                | Y            | Y            | Y            | Y            | Y            | Y            |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md)                                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md)                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| **Advanced SQL Features**                                                                                | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Prepared statement cache](/sql-prepare-plan-cache.md)                                                   | Experimental | Experimental | Experimental | N            | N            | N            |
+| [SQL plan management (SPM)](/sql-plan-management.md)                                                     | Y            | Y            | Y            | N            | N            | N            |
+| [Coprocessor cache](/coprocessor-cache.md)                                                               | Y            | Y            | Experimental | N            | N            | N            |
+| [Placement rules in SQL](/configure-placement-rules.md)                                                  | Experimental | Experimental | Experimental | N            | N            | N            |
```

Comment:
I prefer not to mention it. There is not document for it and we don't recommend users to use it because the SQL interfaces will be changed in the future.